applications/sandbox/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021-2021 Leon Brands. All rights served.
+# License: https://github.com/Legion-Engine/Legion-LLRI/blob/main/LICENSE
+
+project(sandbox LANGUAGES CXX)
+
+file(GLOB_RECURSE source *.hpp *.inl *.cpp)
+add_executable(sandbox ${source})
+
+target_compile_features(sandbox PRIVATE cxx_std_17)
+
+file(GLOB dllDeps ${LLRI_DIR_APPLICATIONS_DEPS}/dll/*.dll)
+
+add_custom_command(TARGET sandbox PRE_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${dllDeps} ${CMAKE_CURRENT_BINARY_DIR}/Debug
+)
+
+add_custom_command(TARGET sandbox PRE_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${dllDeps} ${CMAKE_CURRENT_BINARY_DIR}/Release
+)
```

Comment:
since this is a prebuild step this might be able to use the more robust `file(COPY <source> DESITATION <desitnation>)` command, you could also use `cmake -D` and implement this in a seperate file to be called by the actual build-system in a more controlled manner